keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt

```diff
@@ -82,7 +82,7 @@ class UnhealthyNotificationListener(
     val resourceUrl = "$spinnakerBaseUrl/#/applications/${resource.application}/clusters?${params.toURL()}"
 
     return Notification(
-      subject = "${resource.spec.displayName} is unhealthy",
+      subject = "${resource.spec.displayName} is unhealthy in ${regions.joinToString(" and ")}",
```

Comment:
I suggest making this a little more human by checking the length of the list and using commas for all the but the last list item if the length exceeds 2. so like`us-east-1, us-west-2, and eu-west-1`. If you have access to the spec here as well, might say "all regions" if the length of this list is equal to the length of the regions in the spec.

it's stuff like this that makes UI codebases _so much fun_...

Reply:
Arg. I will attempt this.